pyartifactory/objects.py

```diff
@@ -752,6 +754,32 @@ def delete(self, permission_name: str) -> None:
 class ArtifactoryArtifact(ArtifactoryObject):
     """Models an artifactory artifact."""
 
+    def _walk(
+        self, artifact_path: str, topdown: bool = True, onerror=None
+    ) -> Iterator[ArtifactInfoResponse]:
+        """Iterate over artifact (file or directory) recursively.
+
+        :param artifact_path: Path to file or folder in Artifactory
+        :param topdown: True for a top-down (directory first) traversal
+        :param onerror: Error callback function
+        """
+        info = self.info(artifact_path)
+        if not isinstance(info, ArtifactFolderInfoResponse):
+            yield info
+        else:
+            if topdown:
+                yield info
+            for subdir in [c for c in info.children if c.folder is True]:
```

Comment:
Can you please replace replace the variable c by `child` for more readibility ?

Reply:
Done.